hdfs3/core.py

```diff
@@ -24,7 +24,7 @@ def hdfs_conf(conf_dir=None):
     """ Load HDFS config from default locations. """
     if conf_dir == None:
         confd = os.environ.get('HADOOP_CONF_DIR', os.environ.get('HADOOP_INSTALL',
-                           '') + '/hadoop/conf')
+                           '/etc') + '/hadoop/conf')
```

Comment:
I think after consideration, the warning may be the best option ('Hadoop configuration not found, using localhost:8020 unless otherwise specified"), certainly the easiest. Would that be annoying to anyone?